### PR TITLE
feat: Operator Note Narrative-State Verb Family

### PR DESCRIPTION
**Pull request by sahil-noon**
## Meta

| Change ID | Type | Confidence | Plan | Review |
|-----------|------|------------|------|--------|
| `s2gw` | feat | 4.7/5.0 | 14/14 tasks, 21/21 acceptance ✓ | ✓ 1 cycle |

| Impact | +/− | Net |
|--------|----:|----:|
| raw | +1750 / −19 | +1731 |
| **true** | **+973 / −14** | **+959** |
| └ impl | +442 / −14 | +428 |
| └ tests | +531 / −0 | +531 |

<sub>excludes `fab/`, `docs/` · generated by fab-kit v2.20.9</sub>

**Pipeline:** [intake](https://github.com/sahil87/fab-kit/blob/260823-s2gw-operator-note-narrative-state/fab/changes/260823-s2gw-operator-note-narrative-state/intake.md) ✓ → [apply](https://github.com/sahil87/fab-kit/blob/260823-s2gw-operator-note-narrative-state/fab/changes/260823-s2gw-operator-note-narrative-state/plan.md) ✓ → review ✓ → hydrate ✓ → ship → review-pr

## Summary

The operator has no owned surface for cross-cutting narrative state (phase-plan progress, peer agreements, corrections, merge-gate waits) and resorted to hand-writing keys into the operator state file, violating the Full Mediation doctrine. This change (Phase A of the operator offload plan, Change 1 of 4 in the queue) adds a `fab operator note` verb family over a new `notes:` section in the server-keyed operator state file — the binary owns the mechanical envelope (ids, timestamps, caps, pruning, atomic writes) while the free-prose body stays operator judgment, and Change 2's persisted merge-sequence state depends on its `coordination` note kind.

## Changes

- `src/go/fab/cmd/fab/operator_note.go` (NEW) — the verb family
- `src/go/fab/cmd/fab/operator_state.go` (MODIFY) — skeleton + header
- `src/kit/skills/fab-operator.md` (skill) — new § Notes
- `src/kit/skills/_cli-fab.md` (skill) — signatures
- Tests — `src/go/fab/cmd/fab/operator_note_test.go` (NEW)
- Edge cases in scope (plan § Edge cases, Phase-A subset)

### CODE REVIEW

File: src/go/fab/cmd/fab/operator_note.go [hunk 1]
```diff
@@ -0,0 +1,344 @@
...
+// stale note (updated_at older than noteStaleThreshold) carries the
+// display-only `⚠ <age>` flag in place of the plain age.
+func formatNoteLine(n noteEntry, now time.Time) string {
+	age := formatNoteAge(noteAge(n, now))
+	if noteAge(n, now) > noteStaleThreshold {
+		age = "⚠ " + age
+	}
+	first, _, _ := strings.Cut(n.Text, "\n")
+	return fmt.Sprintf("%s %s %s %s", n.ID, n.Kind, age, first)
+}
```
[the remark this answers is not in the document]
Comment (sahil-noon) [reply src/go/fab/cmd/fab/operator_note.go:132]: Fixed — `formatNoteLine` now renders the documented ` · ` separators (`id · kind · age · first line`), shared by `note list` and the state OPEN NOTES header. (a8ff676)

File: src/go/fab/cmd/fab/operator_note_test.go [hunk 2]
```diff
@@ -0,0 +1,531 @@
...
+	}
+	if !strings.Contains(lines[0], "n1 dependency_wait ⚠ 21d merge-gate wait") {
+		t.Errorf("stale note line wrong: %q", lines[0])
+	}
+	if strings.Contains(lines[0], "second line") {
+		t.Errorf("only the text's first line may render: %q", lines[0])
+	}
+	if !strings.HasPrefix(lines[1], "n2 phase_plan ") || strings.Contains(lines[1], "⚠") {
+		t.Errorf("fresh note line wrong: %q", lines[1])
+	}
```
[the remark this answers is not in the document]
Comment (sahil-noon) [reply src/go/fab/cmd/fab/operator_note_test.go]: Fixed — test assertions updated to the ` · ` format at both the list-line and OPEN-NOTES-header sites. (a8ff676)

File: src/go/fab/cmd/fab/operator_note.go [hunk 3]
```diff
@@ -0,0 +1,344 @@
...
+	if err != nil {
+		return err
+	}
+	notes, err := readNotes(data)
+	if err != nil {
+		return err
+	}
+	all, _ := cmd.Flags().GetBool("all")
+	asJSON, _ := cmd.Flags().GetBool("json")
+
```
[the remark this answers is not in the document]
Comment (sahil-noon) [reply src/go/fab/cmd/fab/operator_note.go:314]: Skipped — deliberate design: this change's plan records `--open` as an accepted explicit no-op spelling of the default (Assumption 11: no `--open`/`--all` conflict error, since erroring on the documented default spelling would punish it). `--all` therefore intentionally wins when combined.
